cpp/tests/copying/copy_tests.cu

```diff
@@ -299,11 +299,11 @@ TYPED_TEST(CopyTestNumeric, CopyIfElseTestScalarColumn)
 
   cudf::numeric_scalar<T> lhs_w(5);
 
-  T rhs[]      = {6, 6, 6, 6};
+  T rhs[]      = {static_cast<T>(6), static_cast<T>(6), static_cast<T>(6), static_cast<T>(6)};
```

Comment:
```suggestion
const auto rhs = cudf::test::make_type_param_vector<TypeParam>({6, 6, 6, 6});
```

Reply:
similar suggestions where multiple `static_cast<T>(` is present.

Reply:
Done.